### PR TITLE
xml-light not compatible with ocaml 5

### PR DESCRIPTION
**Pull request by hhugo**

### CODE REVIEW

File: packages/xml-light/xml-light.2.3/opam [hunk 1]
```diff
@@ -14,6 +14,9 @@ structure, work with it, and print it back to an XML document. It
 support also DTD parsing and checking, and is entirely written in
 OCaml, hence it does not require additional C library."""
 flags: light-uninstall
+depends: [
+  "ocaml" { < "5.0"}
```
Comment (samoht) [inline packages/xml-light/xml-light.2.3/opam:18]: There's already a depends field above :-)

Comment (hhugo) [reply packages/xml-light/xml-light.2.3/opam]: fixed

**Comment by kit-ty-kate** [comment]
@hhugo next time could you paste the error log so it's easier to double-check for everyone and lookup in the future?
```
#=== ERROR while installing xml-light.2.4 =====================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.5.0.0 | file:///home/opam/opam-repository
# path                 ~/.opam/5.0/.opam-switch/build/xml-light.2.4
# command              ~/.opam/opam-init/hooks/sandbox.sh install make install_ocamlfind
# exit-code            2
# env-file             ~/.opam/log/xml-light-7-df3a24.env
# output-file          ~/.opam/log/xml-light-7-df3a24.out
### output ###
# ocamlyacc xml_parser.mly
# ocamlc  xml.mli
# ocamlc  dtd.mli
# ocamlc  xml_parser.mli
# ocamlc  -c xml_parser.ml
# ocamllex xml_lexer.mll
# 248 states, 1238 transitions, table size 6440 bytes
# ocamlc  xml_lexer.mli
# ocamlc  -c xml_lexer.ml
# File "xml_lexer.mll", line 279, characters 25-41:
# Error: Unbound value String.lowercase
# make: *** [Makefile:102: xml_lexer.cmo] Error 2
```

**Comment by kit-ty-kate** [comment]
In any case, many thanks!

**Pull request merged**